kraken-build/src/kraken/std/python/pyproject.py

```diff
@@ -52,9 +52,9 @@ class PackageIndex:
 
 
 @dataclass
-class Pyproject(MutableMapping[str, Any]):
+class TomlConfig(MutableMapping[str, Any]):
```

Comment:
Given that the class no longer has any particular relationship with Python, we should move this into `kraken.std.utils`.